src/ImageFrameWriter.js

```diff
@@ -14,7 +14,13 @@ const ImageFrameWriter = options => {
         await writeStream.write(contentType);
         // await writeStream.write(`Content-Length: ${imageFrame.length}\r\n\r\n`);
         await writeStream.write('\r\n');
-        await writeStream.write(imageFrame);
+
+        // node api accepts writing from Uint8Array or Buffer
+        if(imageFrame instanceof Uint8Array) {
```

Comment:
As per API 
https://nodejs.org/api/stream.html
it should be Uint8Array or buffer.